classes/03class/exercises/c03-docker01/hsy3418/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.11.5
+WORKDIR /tmp
+RUN apk add wget unzip
+RUN wget https://releases.hashicorp.com/terraform/0.12.20/terraform_0.12.20_linux_amd64.zip
+RUN unzip ./terraform_0.12.20_linux_amd64.zip -d /usr/local/bin/
```

Comment:
Good stuff, Heaven. Your final image is around 74MB (less than the 80MB asked in the exercise).

A tip is that you can save further 20MB just by deleting the original zip file downloaded and removing `wget` and `unzip`.

You can also merge the installation process in one docker layer.

````
FROM alpine:3.11.5
WORKDIR /tmp
RUN apk add wget unzip && \
 wget https://releases.hashicorp.com/terraform/0.12.20/terraform_0.12.20_linux_amd64.zip && \
 unzip ./terraform_0.12.20_linux_amd64.zip -d /usr/local/bin/ && \
 rm /tmp/terraform* && \
 apk del wget unzip

CMD ["terraform", "-v"] 
```